scripts/psuedo-init

```diff
@@ -149,7 +149,7 @@ wait_for_ready() {
 main() {
     local short_opts="h:v"
     local long_opts="help,network:,verbose"
-    local getopt_out="" args="$*"
+    local getopt_out="" args="$*" argn="$#"
```

Comment:
This seems unrelated unless you also wanted to augment the debug print before the mount?

Reply:
yeah. i had meant to also log argn. fixing.

Reply:
added argn to the debug.